docs/BR.md

```diff
@@ -1739,142 +1737,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MUST         | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's OCSP responder. |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD       | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's certificate. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.6.8 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be FALSE |
+| `pathLenConstraint` | MUST NOT be present |
+
+**Note**: CAs MUST observe DER encoding rules, such as not explicitly encoding DEFAULT values within OPTIONAL fields.
+
+##### 7.1.2.6.9 Certificate Policies
+
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` |              | The Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)) associated with the given Subscriber Certificate type (see [Section 7.1.2.6.1](#71261-subscriber-certificate-types). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **2**                  | MAY          | The Issuing CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the Issuing CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **3**                  | MUST NOT     | The Issuing CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
+
+If the `policyQualifiers` is permitted and present within a `PolicyInformation` field, it MUST be formatted as follows:
+
+| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| ---                                  | -            | -              | -----         |
+| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | The HTTP URL for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
```

Comment:
"HTTP" should be clarified as "HTTP/HTTPS" so that there is no ambiguity between this qualifier (which discussion in validation-wg indicated HTTPS is allowed/preferred) and the allowed URI schemes for AIA accessDescriptions (which MUST only be "HTTP").

Reply:
Good catch! Fixed!